odk1-src/briefcase-install.rst

```diff
@@ -22,11 +22,16 @@ Setting Up ODK Briefcase
        
        $ java -jar {path/to/Briefcase}
    
+#. Set your local **Briefcase Storage** location.
 
-#. Set the location for Briefcase Storage.
+   .. _briefcase_storage:
```

Comment:
Unless you have a strong reason to do it this way,
I'd probably put the section label above the list item: 

```
.. _briefcase_storage:

#. Set your local **Briefcase Storage** location.
```

That way when someone links to that, the list item header is at the top of the page, instead of scrolled off the top.

(The semantic question is:
is someone linking to the line item, or that specific paragraph?)

Reply:
I put it there because placing it before the list item broke the number sequence. Is there any trick to force it to continue the sequence? (this item would be the third)

Reply:
> is someone linking to the line item, or that specific paragraph?

Yeah, what makes sense is to link to the item, not the paragraph. This needs to be changed

Reply:
I just ran into this exact same problem. What you did seems to be the only solution.